notebooks/ceo_mapping_analysis.ipynb

```diff
@@ -0,0 +1,237 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    meta_dataframe = create_meta_dataframe(path_fn)
</pre><p>Would disagreement_dataframe or disagreement_df be a more fitting name?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/crop-mask/pull/276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>